apps/docs/content/guides/css/text-crop.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Text Crop
+description: A technique to crop the whitespace above and below text to make it appear more centered.
+order: 1
+---
+
+# Text Crop
+
+Every web font includes whitespace above and below the text, while useful in some instances, it can be a problem when vertically centering text or aligning text with other elements. This problem arise when a font has uneven whitespace. This is especially true when using a typeface with a large x-height, like *ABC Favorit*.
```

Comment:
```suggestion
Every web font includes whitespace above and below the text, while useful in some instances, it can be a problem when vertically centering text or aligning text with other elements. This problem arises when a font has uneven whitespace. This is especially true when using a typeface with a large x-height, like *ABC Favorit*.
```